src/vsphere_cpi/lib/cloud/vsphere/vm_config.rb

```diff
@@ -122,7 +135,9 @@ def config_spec_params
       params = {}
       params[:num_cpus] = vm_type.cpu
       params[:memory_mb] = vm_type.ram
+      params[:cpu_allocation] = VimSdk::Vim::ResourceAllocationInfo.new(reservation: @maxMhz * vm_type.cpu) if vm_type.cpu_reserve_full_mhz
```

Comment:
@lnguyen Do we need similar [logic here as we have for memory_reservation_locked_to_max](https://github.com/cloudfoundry/bosh-vsphere-cpi-release/blob/master/src/vsphere_cpi/lib/cloud/vsphere/vm_config.rb#L238-L249), these changes seem related to the GPU support that you added.